src/styles/blocks/nav.scss

```
@@ -0,0 +1,40 @@
+@import "../utils/variables";
+
+.nav {
+  &__list {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+  &__link {
+    display: flex;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 14px;
+    text-transform: uppercase;
+    color: $text-main-color;
+    text-decoration: none;
+    padding: 23px 0;
+    &:hover {
+      color: $button-color;
+    }
+    &.is-active {
+      color: $button-color;
+      position: relative;
+    }
+    &.is-active::after {
+      content: "";
+      position: absolute;
+      bottom: 0;
+      width: 37px;
+      height: 4px;
+      background: $button-color;
+      border-radius: 8px;
+    }
+  }
+}
```

Add empty lines between logical blocks

```suggestion
.nav {
  &__list {
    display: flex;
    justify-content: space-between;
    list-style: none;
    padding: 0;
    margin: 0;
  }
  
  &__item:not(:last-child) {
    margin-right: 20px;
  }
  
  &__link {
    display: flex;
    font-size: 12px;
    font-weight: 500;
    line-height: 14px;
    text-transform: uppercase;
    color: $text-main-color;
    text-decoration: none;
    padding: 23px 0;
    
    &:hover {
      color: $button-color;
    }
    
    &.is-active {
      color: $button-color;
      position: relative;
    }
    
    &.is-active::after {
      content: "";
      position: absolute;
      bottom: 0;
      width: 37px;
      height: 4px;
      background: $button-color;
      border-radius: 8px;
    }
  }
}
```

---

Done